scripts/linters/general_purpose_linter.py

```diff
@@ -293,14 +284,27 @@
     {
         'regexp': re.compile(r'\$parent'),
         'message': 'Please do not access parent properties ' +
-                   'using $parent. Use the scope object' +
+                   'using $parent. Use the scope object ' +
                    'for this purpose.',
         'excluded_files': (),
         'excluded_dirs': ()
     }
 ]
 
 BAD_PATTERNS_PYTHON_REGEXP = [
+    {
+        'regexp': re.compile(r'__author__'),
+        'message': 'Please remove author tags from this file.',
+        'excluded_files': (),
+        'excluded_dirs': ()
+    },
+    {
+        'regexp': re.compile(r'datetime.datetime.now\(\)'),
```

Comment:
Ditto?

Reply:
This is a python method, before linter is checking it in every file but now it will be checked in python files only.

Reply:
before it is checking all the files including HTML, CSS, TS etc, which is taking extra time, now it will only check python file.